tests/components/matrix/test_send_message.py

```diff
@@ -53,6 +78,28 @@ async def test_send_message(
     for room_alias_or_id in TEST_JOINABLE_ROOMS:
         assert f"Message delivered to room '{room_alias_or_id}'" in caplog.messages
 
+    # Send bot notice (m.notice) message.
+    matrix_bot._handle_multi_room_send.reset_mock()
```

Comment:
Can we make a parameterized test out of it as this test tests the three different message formats

Reply:
👌  Yes, I can refactor those tests.

Reply:
:heavy_check_mark: done